examples/translator/common/src/lib.rs

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+pub mod proto;
+
+use log::{info, warn};
+use proto::translator::{TranslateRequest, TranslateResponse};
+
+/// Client for a translator Node.
+pub struct TranslatorClient {
+    invocation_sender: oak::io::Sender<oak::grpc::Invocation>,
+}
+
+// TODO(#549): autogenerate most of this code based on the gRPC service definition.
+impl TranslatorClient {
+    /// Attempt to create a new translator Node and connection to it, assuming
+    /// that the Wasm code for the translator Node is available under the given
+    /// config name.  Returns `None` if it was not possible to create the
+    /// translator.
+    pub fn new(config_name: &str) -> Option<TranslatorClient> {
+        let (wh, rh) = oak::channel_create().expect("failed to create channel");
```

Comment:
Could use the new fancy `oak::io::channel_create::<Invocation>` here too?

Reply:
Done.